Add he_il.json

idodana opened the pull request.

idodana committed:
Add most of he_il.json

idodana committed:
Add he_il.json

CAS-ual-TY commented:
First of all, thank you @idodana 
Now, I dont speak hebrew so I will just have to trust you on this one.

A question, I see that you have not translated everything. For some nodes this definitely makes sense, especially the logic gates. But what about:
- Paralellize
- De-Parallelize
- Multi Constant Node
- Multi Equals Branch

Can you just make sure that these are best as they are (or also translate them)?
Thank you!

CAS-ual-TY committed:
Fixed whitespaces, tabs and new-lines

These changes are already present in master but not commited yet

idodana commented:
I am not sure how to translate these four strings. There seems to be no Hebrew translation of them. Still, I will keep searching for translations, and if I find one I'll add it.

By the way, I really like your mod.

CAS-ual-TY committed:
Removed old "(!)" language concept

According to https://github.com/CAS-ual-TY/Visibilis/commit/6835fb2b6ec75df6b8ecbcc0e0dd2c664ead2dc8

CAS-ual-TY commented:
Alright! Do explain these things to you:

**Parallelize/De-Parallelize**
I kinda made these words up. They describe a concept which (again) I crafted myself. To explain it, I made a professional Paint graphic:
![picture](https://user-images.githubusercontent.com/41904979/85951263-57d91700-b962-11ea-90eb-1aa5711d38cd.png)
I hope that clears it up. You can come up with something that in your opinion fits best.

**Multi Constant Node**
The Multi Constant Node has mainly just been a test for dynamic data types. The Output types depend on the Input types. It does not do anything other than give out what comes in (so nothing). You can only really use it for organization of your code prints.

**Multi Equals Branch**
Simple way of comparing multiple objects of any type. Example:
![2020-06-28_17 15 24](https://user-images.githubusercontent.com/41904979/85951389-42b0b800-b963-11ea-8d0d-c811f24afb6b.png)
This would print out:
1st
2nd

--

Finally, I just finished a major rewrite of a system which has a few localizations (which is why I committed to your repo again). Could you quickly fix them again? That would be all then (Ill probably merge even if you dont do anything :P)
Changes of Jun 28th:
https://github.com/CAS-ual-TY/Visibilis/commits/master/src/main/resources/assets/visibilis/lang/en_us.json

Thanks and have a nice day!
